DashboardEmbedding/README.md

```diff
@@ -76,11 +76,11 @@ This sample demonstrates the embedding the dashboard with iFrame based Single Si
 
   * Run your `EmbedDashboardswithSSOusingToken` sample.
 
-> **NOTE:** The default behavior is to display `iframe` content in the `Chrome` browser. If you do not have `Chrome`, please manually retrieve the `iframe` content from the application.
+    ![dashboard image](https://github.com/boldbi/blazor-webassembly-sample/assets/91586758/9719d6e4-f0e0-4254-afc6-051489550eb6)
```

Comment:
Why using blazor webassembly images?

Reply:
All samples readme file image used. So, this sample also included. Previously image included. Now note content and image place changed.

Reply:
Ok. But this should be changed in future with separate PR for all Github sample have own assets.

Reply:
Okay